docs/topics/AutoType.adoc

```diff
@@ -12,21 +12,27 @@ WARNING: Auto-Type will be disabled when run with a Wayland compositor on Linux.
 === Configure Global Auto-Type
 You can define a global Auto-Type hotkey that starts the Auto-Type process. To configure the hotkey, perform the following steps:
 
-1. Navigate to _Tools_ -> _Settings_ -> Auto-Type tab *(1)*. Click into the _Global Auto-Type shortcut_ box and press the desired key combination that will trigger the Auto-Type process *(2)*.
-+
+Navigate to _Tools_ -> _Settings_ -> Auto-Type tab *(1)*. Click into the _Global Auto-Type shortcut_ box and press the desired key combination that will trigger the Auto-Type process *(2)*.
+
 .Auto-Type settings
 image::autotype_settings.png[]
-+
+
 You can configure additional Auto-Type settings in this window such as start delay, inter-key typing delay, and matching options. If Auto-Type is not working well for you, try adjusting the default delays.
 
+A new feature has been added to remember the last used entry for a short period of time between presses of the global Auto-Type hotkey. This means you can type different sequences or parts of sequences during a login workflow without having to find the specific entry again.

```

Comment:
Shouldn't the documentation just document what the current state is instead of highlighting new features? Just asking :sweat_smile:

Reply:
Good point, this was one of my first changes and I stopped doing that elsewhere.